capstone/src/main/webapp/index.html

```diff
@@ -22,14 +22,65 @@ <h1>Enforcer</h1>
       </div>
     </div>
     <div id="content" class="content">
+        <div id="info-box">
+            <h2 color="#19B3B1">Click on a Zipcode to See Associated Police Precincts</h2>
+            <p id="zipcode">Zipcode: none selected</p>
+            <p id="neighborhood">Neighborhood: none selected</p>
+            <p id="precincts">Associated Precinct(s): none selected</p>
+        </div>
       <div id="map-key">
-        <h2>Check to See Precinct Data</h2>
+        <h2>Click to See Precinct Data</h2>
         <input type="checkbox" id="sentimentCheck" name="check" onclick="onlyOne(this)">
-        <label for="sentimentCheck">Show sentiment</label><br>
-        <input type="checkbox" id="crimerateCheck" name="check" onclick="onlyOne(this)">
-        <label for="crimerateCheck">Show crime rate</label><br>
-        <input type="checkbox" id="incomeCheck" name="check" onclick="onlyOne(this)">
-        <label for="incomeCheck">Show household incomes</label><br><br>
+        <label for="sentimentCheck">Show survey sentiments of July by police precinct</label><br>
+        <input type="checkbox" id="weekCheck" name="check" onclick="onlyOne(this)">
+        <label for="crimerateCheck">Show sentiment by week</label>
+        <input type="checkbox" id="dayCheck" name="check" onclick="onlyOne(this)">
+        <label for="incomeCheck">Show sentiment by day</label><br>
+        <label for="timeframes">Choose a timeframe:</label>
+        <select name="timeframes" id="timeframe">
+                <option value="all" id="all">All</option>
+            <optgroup label="Weeks" id="weeks">
+                <option value="Week1">July 1-7</option>
+                <option value="Week2">July 8-14</option>
+                <option value="Week3">July 15-21</option>
+                <option value="Week4">July 22-28</option>
+                <option value="Week4">July 29-31</option>
+                </optgroup>
```

Comment:
You might want to align the indents for the start brace of `</optgroup>` to improve readability.